pkg/api/dtos/plugins.go

```diff
@@ -53,10 +53,10 @@ func (slice PluginList) Swap(i, j int) {
 }
 
 type ImportDashboardCommand struct {
-	PluginId  string                         `json:"pluginId"`
-	Path      string                         `json:"path"`
-	Overwrite bool                           `json:"overwrite"`
-	Dashboard *simplejson.Json               `json:"dashboard"`
-	Inputs    []plugins.ImportDashboardInput `json:"inputs"`
-	FolderId  int64                          `json:"folderId"`
+	PluginId  string                         `json:"pluginId" binding:"Required"`
+	Path      string                         `json:"path" binding:"Required"`
+	Overwrite bool                           `json:"overwrite" binding:"Required"`
+	Dashboard *simplejson.Json               `json:"dashboard" binding:"Required"`
+	Inputs    []plugins.ImportDashboardInput `json:"inputs" binding:"Required"`
+	FolderId  int64                          `json:"folderId" binding:"Required"`
```

Comment:
I think that only if `Dashboard` is missing it will result to segmentation fault because all the others would get the zero value of each type. Therefore I would suggest that only `Dashboard` should be required.

Reply:
Agree make sense

Reply:
Oh yeah, I guess that does make sense.

Reply:
@papagian @marefr Validity wise though, which fields do we require when these commands are sent? Segfaults aren't the only possible bugs after all.

Reply:
You can have a look at the implementation of the import: https://github.com/grafana/grafana/blob/master/pkg/plugins/dashboard_importer.go#L46

Reply:
Think it should be enough

Reply:
Hard for me to say which are required. Therefore, I would expect that such error cases would be handled lately when the specific fields are actually used. And if there are some missing error checks we will fix them in the future as soon as we discover them. But that's only my point of view, therefore I don't insist.

Reply:
I agree with Sofia. @aknuds1 Will you make the needed changes?

Reply:
@papagian @marefr OK, thanks, I'll take care of it.